### PR TITLE
Create github templates

### PR DESCRIPTION
**Pull request by CodeTheory**
- [x] Issue Template
- [x] Pull Request Template

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!--
+  Hi there! Thank you for discovering and submitting an issue. 
+
+  Before you submit this; let's make sure of a few things. 
+  Please make sure the following boxes are ticked if they are correct.
+  If not, please try and fulfil these first.
+-->
+
+- [ ] I am on the [latest](github.com/zeit/hyper/releases/latest) Hyper.app version
```
Comment (matheuss) [inline .github/issue_template.md]: The link is broken – you need `https://`

**Comment by CodeTheory** [comment]
Updated @matheuss

**Comment by CodeTheory** [comment]
Okay, now I'm done.

**Comment by matheuss** [comment]
Looks 💯 to me

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+<!-- 
+  Once those are done, if you're able to fill in the following list with your information,
+  it'd be very helpful to whoever handles the issue.
+-->
+
+- **OS version and name**: <!-- Replace with version + name -->
+- **Hyper.app version**: <!-- Replace with version -->
+- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: <!-- Gist Link Here -->
+- **Relevent information from devtools** _(CMD+SHIFT+I)_: <!-- Replace with info if applicable, or N/A -->
```
Comment (CodeTheory) [inline .github/issue_template.md]: @fearphage is it `CTRL+SHIFT+I` for other systems?

**Pull request merged**